dipy/reconst/csdeconv.py

```diff
@@ -775,8 +809,9 @@ def auto_response(gtab, data, roi_center=None, roi_radius=10, fa_thr=0.7,
         radius of cubic ROI
     fa_thr : float
         FA threshold
-    fa_operator : lambda
-        operator used to compare the FA with the fa_thr.
+    fa_operator : function
+        Function that defines operator used to compare the FA with the fa_thr.
+        Must be defined as `def function_name(FA, fa_thr)` and return a bool array.
```

Comment:
why? I believe lambdas and functions are the same in python (expect lambdas don't get a function name).